generate/input/descriptor.json

```diff
@@ -2918,6 +2915,9 @@
     "transfer_progress": {
       "dupFunction": "git_transfer_progress_dup"
     },
+    "push_transfer_progress": {
```

Comment:
I don't believe you need this block. There is no git_push_transfer_progress struct, and it is also theoretically would not compile if that struct was passed to the dupFunction you declared here.